src/ucp/tag/rndv.c

```diff
@@ -715,15 +709,17 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_matched, (worker, rreq, rndv_rts_hdr),
             }
         }
         /* put protocol is allowed - register receive buffer memory for rma */
+        ucs_assert(rndv_rts_hdr->size <= rreq->recv.length);
```

Comment:
why assert? this should be MESSAGE_TRUNCATED error

Reply:
it's checked before this (line 676), so here it has to be always `true`

Reply:
ok